src/test/java/io/github/acm19/aws/interceptor/test/AmazonOpenSearchServiceSample.java

```diff
@@ -89,7 +89,8 @@ public static void main(final String[] args) throws IOException, ParseException
     }
 
     private void makeRequest() throws IOException {
-        // todo: https://github.com/acm19/aws-request-signing-apache-interceptor/issues/95
+        // Remove when get support is added to OpenSearch Serverless
```

Comment:
Shouldn't this be a TODO: 

get -> GET
add a period

Reply:
I'd rather not have TODO comments at all, therefore enforcing it in checkstyle. I think we don't even need this comment. Fixed the other stuff. BTW, do you have a ticket to where to track GET support for the OpenSearch Serverless project?

Reply:
Thanks. Let me find out if I am allowed to put the number here.